CalibTracker/SiPixelESProducers/src/module.cc

```diff
@@ -0,0 +1,6 @@
+#include "DataFormats/SiPixelDetId/interface/PixelFEDChannel.h"
+#include "FWCore/Utilities/interface/typelookup.h"
+
+typedef std::unordered_map<std::string,PixelFEDChannelCollection> PixelFEDChannelCollectionMap;
+
+TYPELOOKUP_DATA_REG(PixelFEDChannelCollectionMap);
```

Comment:
The framework group strongly suggests not declaring an EventSetup type from the same package as the Producer.  The reason is this requires all consumers of the type to link to the implementations library. Instead, the types should be declared in the same library which holds the header file for the type. In this case it should be done in DataFormats/SiPixelDetId. This would not be the first DataFormats package to do so.